salt/metalk8s/orchestrate/solutions/deploy-components.sls

```diff
@@ -37,16 +37,16 @@ include:
   {%- for crd_file in crd_files %}
     {%- set sls_content = salt.saltutil.cmd(
             tgt=pillar.bootstrap_id,
-            fun='slsutil.renderer',
+            fun='metalk8s.file_read',
             kwarg={
                 'path': crd_file,
-                'default_renderer': renderer,
             },
           )[pillar.bootstrap_id]['ret'] %}
+
 {{ action }} CRD "{{ crd_file }}" for Solution {{ solution.name }}:
-  module.run:
-    - state.template_str:
-      - tem: "{{ sls_content | yaml }}"
+  {{ state }}:
+    - manifest:
+        {{ sls_content | yaml(False) | indent(8) }}"
```

Comment:
What is the use of `False` here?

Reply:
It corresponds to the `default_flow_style=False` argument of the `yaml.dump()` function in Python.